src/Assets/TestProjects/AppThatOutputsDotnetLaunchProfile/Program.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace MSBuildTestApp
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var value = Environment.GetEnvironmentVariable("DOTNET_LAUNCH_PROFILE");
+            Console.WriteLine($"DOTNET_LAUNCH_PROFILE=<<<{value}>>>");
```

Comment:
What is the <<< and >>> for? Is that to separate the output so it's more clear it is the string coming from the console?

Reply:
Its for the Contains check in the test assertion. I can't just assert on the presence of ```DOTNET_LAUNCH_PROFILE=``` for a null/empty result.